FirebaseMessaging/Tests/UnitTests/FIRMessagingAuthKeychainTest.m

```diff
@@ -15,8 +15,8 @@
  */
 
 #import <TargetConditionals.h>
-#if !TARGET_OS_MACCATALYST && !TARGET_OS_IOS
-// Skip keychain tests on Catalyst and iOS
+#if !TARGET_OS_MACCATALYST && !SWIFT_PACKAGE
```

Comment:
Is MacOS ok now? I suspect this should be iOS, tvOS and not Catalyst, not macOS, not Swift Package.

Reply:
this builds on non swift package, when use swift package, it failed at least for iOS and tvOS locally. I'm guessing CI wasn't running tvOS that's why it was passing for previous change.